src/site/content/en/mini-apps/what-are-super-apps/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: What are super apps?
```

Comment:
I think we need a title that makes this page feel more connected to mini-apps.

When I look at the index page, it seems weird that the first item is super apps:

![image](https://user-images.githubusercontent.com/4713486/107722114-1861ff80-6c93-11eb-9f5c-526ecd6c1554.png)

As you mention in the intro, to understand mini apps you need to first understand super apps. Can you concisely convey that idea in the title somehow?

Reply:
"For mini apps you need super apps"